CMakeLists.txt

```diff
@@ -118,19 +121,51 @@ write_basic_package_version_file(${CMAKE_CURRENT_BINARY_DIR}/${PN}ConfigVersion.
                                  COMPATIBILITY AnyNewerVersion)
 
 # Install our files
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${PN}Config.cmake
-              ${CMAKE_CURRENT_BINARY_DIR}/${PN}ConfigVersion.cmake
-        DESTINATION ${CMAKECONFIG_INSTALL_DIR})
-install(EXPORT "${PN}Targets"
-        NAMESPACE "${PN}::"
-        DESTINATION ${CMAKECONFIG_INSTALL_DIR})
+if(NOT(${INSTALL_PYMOD} AND ${NATIVE_PYTHON_INSTALL}))
+    install(FILES ${CMAKE_CURRENT_BINARY_DIR}/gau2grid.h
+                  ${CMAKE_CURRENT_BINARY_DIR}/gau2grid_pragma.h
+                  DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/${PN})
+
+    install(TARGETS gg
+            EXPORT "${PN}Targets"
+            ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+            LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR})
+
+    install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${PN}Config.cmake
+                  ${CMAKE_CURRENT_BINARY_DIR}/${PN}ConfigVersion.cmake
+            DESTINATION ${CMAKECONFIG_INSTALL_DIR})
+    install(EXPORT "${PN}Targets"
+            NAMESPACE "${PN}::"
+            DESTINATION ${CMAKECONFIG_INSTALL_DIR})
+endif()
 
 if(${INSTALL_PYMOD})
+    if(${NATIVE_PYTHON_INSTALL})
+        execute_process(COMMAND ${PYTHON_EXECUTABLE} -c
+                        "import sys; print(sys.prefix);"
+                        OUTPUT_VARIABLE CMAKE_INSTALL_PREFIX
+                        OUTPUT_STRIP_TRAILING_WHITESPACE)
+        execute_process(COMMAND ${PYTHON_EXECUTABLE} -c
+                        "from distutils import sysconfig as s; import os; import sys; cmake_install_prefix = sys.prefix; prefix_lib = s.get_config_var('LIBDIR'); print(prefix_lib.replace(os.path.commonpath([prefix_lib, cmake_install_prefix]), '').strip('/'));"
+                        OUTPUT_VARIABLE CMAKE_INSTALL_LIBDIR
+                        OUTPUT_STRIP_TRAILING_WHITESPACE)
+        execute_process(COMMAND ${PYTHON_EXECUTABLE} -c
+                        "from distutils import sysconfig as s; import os; prefix_lib = s.get_config_var('LIBDIR'); spdir = s.get_python_lib(plat_specific=True); print(spdir.replace(os.path.commonpath([prefix_lib, spdir]), ''));"
+                        OUTPUT_VARIABLE PYMOD_INSTALL_LIBDIR
+                        OUTPUT_STRIP_TRAILING_WHITESPACE)
+    endif()
+
+    execute_process(COMMAND ${PYTHON_EXECUTABLE} -c
+                    "from numpy import distutils; print(distutils.misc_util.get_shared_lib_extension(is_python_ext=False))"
+                    OUTPUT_VARIABLE PYLIB_EXTENSION
+                    OUTPUT_STRIP_TRAILING_WHITESPACE)
+
```

Comment:
Well, this makes this hard to build a template for and distribute.

Reply:
hmm? the custom command calls won't be varying.

or rather, the need for `CMAKE_PREFIX_PATH`, `CMAKE_INSTALL_LIBDIR`, `PYMOD_INSTALL_LIBDIR` is b/c the last is my scheme for getting "pythonxxx/site-packages/" into the install path. Possibly if [`GNUINstallDirs`](https://github.com/Kitware/CMake/blob/master/Modules/GNUInstallDirs.cmake) allows "/" in variable vals (and again only installing pymodule -- notice that I had to suppress installing other components), then only the first two would be needed, and those are cmake std (nothing to do with my habits).

or could pull in https://github.com/psi4/psi4/blob/master/cmake/FindPythonLibsNew.cmake, which we got from pb11, which they got from numpy or similar project. That too sets a site-packages dir through a similar route. I went with commonpath directly b/c I needed paths subtracted and that's easier in py.

Reply:
I guess if they will not vary then it might work, but one main concern is the complexity is higher than I would have hoped for. I wonder if checking out `scikit-build` again is worthwhile.